app/assets/stylesheets/curation_concerns/_colors.scss

```diff
@@ -0,0 +1,7 @@
+// Colors
```

Comment:
Can you add a comment here about why we've chosen these colors over the defaults?